internal/start/start.go

```diff
@@ -906,3 +903,17 @@ func ExcludableContainers() []string {
 	}
 	return names
 }
+
+func formatMapForEnvconfig(input map[string]string, output *bytes.Buffer) {
```

Comment:
```suggestion
func formatMapForEnvConfig(input map[string]string, output *bytes.Buffer) {
```
 Nitpick

Reply:
Hi! @sweatybridge
I'm trying to see the difference between the line in the PR and the suggested change, but I cant find it.

Reply:
env **C** onfig - capitalized `C` :)

Reply:
It seems I need glasses, lol.

Reply:
@sweatybridge @kamilogorek
I've just changed the function name and replaced all occurrences.